db/migrate/20220902135819_project_and_pricelist_items_change_types.rb

```diff
@@ -0,0 +1,47 @@
+class ProjectAndPricelistItemsChangeTypes < ActiveRecord::Migration[6.1]
+  def up
+    change_column 'ratemanagement.projects', :id, :integer, limit: 4, null: false
```

Comment:
squash changes into `db/migrate/20220816081007_create_rate_management_pricelists.rb` and `db/migrate/20220810115417_create_rate_management_projects_table.rb`